pom.xml

```diff
@@ -1723,37 +1723,12 @@
                         <configuration combine.children="merge">
                             <compilerArgs>
                                 <arg>-XDcompilePolicy=simple</arg>
-                                <!-- TODO enable more checks, i.e. remove "-XepDisableAllChecks" -->
                                 <arg>
                                     -Xplugin:ErrorProne
                                     -XepExcludedPaths:.*/target/generated-(|test-)sources/.*
-                                    -XepDisableAllChecks
-                                    -Xep:ArrayEquals:ERROR
-                                    -Xep:ArrayHashCode:ERROR
-                                    -Xep:ArrayToString:ERROR
-                                    -Xep:ArraysAsListPrimitiveArray:ERROR
-                                    -Xep:BadInstanceof:ERROR
-                                    -Xep:BoxedPrimitiveConstructor:ERROR
-                                    -Xep:ClassCanBeStatic:ERROR
-                                    -Xep:CompareToZero:ERROR
-                                    -Xep:ComparingThisWithNull:ERROR
-                                    -Xep:EqualsIncompatibleType:ERROR
-                                    -Xep:FallThrough:ERROR
-                                    -Xep:FormatString:ERROR
-                                    -Xep:GetClassOnClass:ERROR
-                                    -Xep:ImmutableSetForContains:ERROR
-                                    -Xep:InconsistentHashCode:ERROR
-                                    -Xep:InjectOnConstructorOfAbstractClass:ERROR
-                                    -Xep:MissingOverride:ERROR
-                                    -Xep:NullOptional:ERROR
-                                    -Xep:ObjectToString:ERROR
-                                    -Xep:OptionalEquality:ERROR
-                                    -Xep:OptionalMapUnusedValue:ERROR
```

Comment:
This disables checks that are not enabled by default on the ERROR level

Reply:
Ha, this is what I suspected. I was hoping all were ERROR by default; I'll find which is which, thanks.

Reply:
I re-enabled those checks, thanks!